### PR TITLE
 update alpine version & fix a grammer mistake

### PR DESCRIPTION
**Pull request by jonyhy96**
update alpine version & fix a grammer mistake
```release-note
 None
```

### CODE REVIEW

**Comment by jonyhy96** [comment]
lgtm

File: docs/user-journeys/users/application-developer/advanced.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ You may also find the following info helpful:
...
 
-If you do not have a {% glossary_tooltip text="cluster operator" term_id="cluster-operator" %} to manage the installation of Service Catalog, you can do so using [Helm](/docs/tasks/service-catalog/install-service-catalog-using-helm/){:target="_blank"} or an [installer binary](/docs/tasks/service-catalog/install-service-catalog-using-sc/){:target="_blank"}.
+If you do not have a {% glossary_tooltip text="cluster operator" term_id="cluster-operator" %} to manage the installation of Service Catalog, you can do so using [Helm](/docs/tasks/service-catalog/install-service-catalog-using-helm/){:target="_blank"} or a [installer binary](/docs/tasks/service-catalog/install-service-catalog-using-sc/){:target="_blank"}.
```
Comment (zhangxiaoyu-zidif) [inline docs/user-journeys/users/application-developer/advanced.md]: installer [ɪns'tɔ:lər]
so `an` seems to be right.
/cc @heckj

Comment (jonyhy96) [reply docs/user-journeys/users/application-developer/advanced.md]: its my fault,Thanks for reply
@zhangxiaoyu-zidif

**Pull request closed without merging**